pkgs/top-level/all-packages.nix

```diff
@@ -10007,20 +10007,30 @@ with pkgs;
 
   nodenv = callPackage ../development/tools/nodenv { };
 
+  # The NodeJS packages has to be build in 32bit environment if host platform is
+  # also 32bit because it uses 32bit stubs and links against 32bit OpenSSL. The
+  # only architecture that generally supports execution of 32bit is x86_64 and
+  # thus that is the only one handled here.
+  callPackageNodejs =
```

Comment:
This is a public attribute that people may use even though it’s an internal implementation detail. I think it makes sense to move it to the `v{18,20,22,…}.nix` files (or at least avoid exposing it).

Reply:
That was a previous implementation that I changed on bases of these arguments https://github.com/NixOS/nixpkgs/pull/220204#pullrequestreview-1347419337.

Reply:
>the standard callPackage is publicly available
Yes, because it is a generic public interface while `callPackageNodejs` is an alias used only to build Node.js package. Is there some other use case I’m missing here?

Reply:
Like packaging a custom version in overlays?

Reply:
Isn’t `override{,Attrs}` enough in this case?

Reply:
You are right.

I will move it to the vxx.nix files.